zone/lua_client.cpp

```diff
@@ -19,3011 +18,484 @@
+					 (void(Lua_Client::*)(uint32, int))&Lua_Client::AddEXP,
+					 (void(Lua_Client::*)(uint32, int, bool))&Lua_Client::AddEXP);
+	client["AddExpeditionLockout"] = sol::overload(
+	    (void(Lua_Client::*)(std::string, std::string, uint32))&Lua_Client::AddExpeditionLockout,
+	    (void(Lua_Client::*)(std::string, std::string, uint32, std::string))&Lua_Client::AddExpeditionLockout);
+	client["AddExpeditionLockoutDuration"] = sol::overload(
+	    (void(Lua_Client::*)(std::string, std::string, int))&Lua_Client::AddExpeditionLockoutDuration,
+	    (void(Lua_Client::*)(std::string, std::string, int, std::string))&Lua_Client::AddExpeditionLockoutDuration);
+	client["AddItem"] = (void(Lua_Client::*)(sol::table))&Lua_Client::AddItem;
+	client["AddLDoNLoss"] = (void(Lua_Client::*)(uint32))&Lua_Client::AddLDoNLoss;
+	client["AddLDoNWin"] = (void(Lua_Client::*)(uint32))&Lua_Client::AddLDoNWin;
+	client["AddLevelBasedExp"] =
+	    sol::overload((void(Lua_Client::*)(int))&Lua_Client::AddLevelBasedExp,
+			  (void(Lua_Client::*)(int, int))&Lua_Client::AddLevelBasedExp,
+			  (void(Lua_Client::*)(int, int, bool))&Lua_Client::AddLevelBasedExp);
+	client["AddMoneyToPP"] =
+	    sol::overload((void(Lua_Client::*)(uint32, uint32, uint32, uint32))&Lua_Client::AddMoneyToPP,
+			  (void(Lua_Client::*)(uint32, uint32, uint32, uint32, bool))&Lua_Client::AddMoneyToPP);
+	client["AddPlatinum"] = sol::overload((void(Lua_Client::*)(uint32))&Lua_Client::AddPlatinum,
+					      (void(Lua_Client::*)(uint32, bool))&Lua_Client::AddPlatinum);
+	client["AddPVPPoints"] = (void(Lua_Client::*)(uint32))&Lua_Client::AddPVPPoints;
+	client["AddSkill"] = (void(Lua_Client::*)(int,int))&Lua_Client::AddSkill;
+	client["Admin"] = (int(Lua_Client::*)(void))&Lua_Client::Admin;
+	client["AssignTask"] = sol::overload((void(Lua_Client::*)(int))&Lua_Client::AssignTask,
+					     (void(Lua_Client::*)(int, int))&Lua_Client::AssignTask,
+					     (void(Lua_Client::*)(int, int, bool))&Lua_Client::AssignTask);
+	client["AssignToInstance"] = (void(Lua_Client::*)(int))&Lua_Client::AssignToInstance;
+	client["AutoSplitEnabled"] = (bool(Lua_Client::*)(void))&Lua_Client::AutoSplitEnabled;
+	client["BreakInvis"] = (void(Lua_Client::*)(void))&Lua_Client::BreakInvis;
+	client["CalcATK"] = &Lua_Client::CalcATK;
+	client["CalcCurrentWeight"] = &Lua_Client::CalcCurrentWeight;
+	client["CalcPriceMod"] = (float(Lua_Client::*)(Lua_Mob,bool))&Lua_Client::CalcPriceMod;
+	client["CanHaveSkill"] = (bool(Lua_Client::*)(int))&Lua_Client::CanHaveSkill;
+	client["CashReward"] = &Lua_Client::CashReward;
+	client["ChangeLastName"] = (void(Lua_Client::*)(std::string))&Lua_Client::ChangeLastName;
+	client["CharacterID"] = (uint32(Lua_Client::*)(void))&Lua_Client::CharacterID;
+	client["CheckIncreaseSkill"] =
+	    sol::overload((void(Lua_Client::*)(int, Lua_Mob))&Lua_Client::CheckIncreaseSkill,
+			  (void(Lua_Client::*)(int, Lua_Mob, int))&Lua_Client::CheckIncreaseSkill);
```

Comment:
Lots of inconsistent alignment due to tabs in this file (I don't use tab width 4). We should probably standardize sol::overload registrations with each overload on its own line with a single indent

Reply:
I just used clang-format to format these